bot/controllers/verify.js

```diff
@@ -0,0 +1,152 @@
+const models = require('../../db/models');
+const uuidv4 = require('uuid/v4');
+const util = require('apex-util');
+const nodemailer = require('nodemailer');
+require('dotenv').config();
```

Comment:
envs should be added to max.config.js. Loading in the env file like this is redundant.

Reply:
Fixed!

Reply:
Thank you, the env file logic that was implemented has been changed.